src/components/pageHeader.jsx

```diff
@@ -23,6 +40,11 @@ const PageHeader = ({ isDao, header, headerEl, customTerms }) => {
         >
           {customTerms ? getTerm(customTerms, header) : header}
         </Box>
+        {isDao && (
+          <Button onClick={handleEditProfile} mr={5} variant='outline'>
+            Edit 3Box Profile
```

Comment:
Do we really want this on all DAO pages?

If so, could we just change this to 'Edit Profile'

Reply:
just updated this on the latest commit #693097f